runtimes/core/src/api/reqauth/meta.rs

```diff
@@ -78,7 +78,8 @@ impl MetaMapMut for reqwest::header::HeaderMap {
 
 impl MetaMap for axum::http::HeaderMap {
     fn get_meta(&self, key: MetaKey) -> Option<&str> {
-        self.get(key.header_key()).and_then(|v| v.to_str().ok())
+        self.get(key.header_key())
```

Comment:
Can you add a comment explaining why this is necessary?

Reply:
added! let me know if its not explaining it well enough